ledger-obsolete/src/main/kotlin/net/corda/v5/ledger/obsolete/LedgerGroupParameters.kt

```diff
@@ -1,10 +1,19 @@
 @file:JvmName("LedgerGroupParameters")
+
 package net.corda.v5.ledger.obsolete
 
 import net.corda.v5.membership.GroupParameters
-import net.corda.v5.membership.getValue
 
-const val NOTARIES_KEY = "corda.notaries"
+private const val NOTARIES_KEY = "corda.notary"
```

Comment:
I would expect key to still be plural form as we allow a list of notaries in the network